cmd/prometheus.go

```diff
@@ -37,7 +38,7 @@ func RunPrometheusHTTPHandler(connectionString string) (net.Listener, *http.Serv
 		logrus.WithField("connection_string", connectionString).Infoln("Start prometheus http handler")
 		err := server.Serve(listener)
 		if err != nil {
-			logrus.WithError(err).Errorln("Error from http server that process prometheus metrics")
+			logrus.WithField(logging.FieldKeyEventCode, logging.EventCodeErrorPrometheusHTTPHandler).WithError(err).Errorln("Error from http server that process prometheus metrics")
```

Comment:
I saw that you prefer HTTP now :)

```suggestion
			logrus.WithField(logging.FieldKeyEventCode, logging.EventCodeErrorPrometheusHTTPHandler).WithError(err).Errorln("Error from HTTP server that process prometheus metrics")
```